session_pool.go

```diff
@@ -287,6 +287,18 @@ func (pool *SessionPool) ExecuteAndCheck(q string) (*ResultSet, error) {
 	return rs, nil
 }
 
+func (pool *SessionPool) ShowSpaces() ([]LabelName, error) {
```

Comment:
its ambiguous to reuse the LabelName, can we redefine a struct SpaceName?

Reply:
done